framework/src/application/storage/migrations/node/20200226000000_add_nonce_to_trs.sql

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+ALTER TABLE "trs" ADD COLUMN IF NOT EXISTS "nonce" TEXT NOT NULL DEFAULT '0';
+ALTER TABLE "trs" ALTER COLUMN "nonce" DROP DEFAULT;
```

Comment:
why are we adding and removing the default?

Reply:
We removed `default` so every transaction must have its own value, so there may not be any transaction mistakenly added without nonce, DB will catch it.